### DISCUSSION TITLE
非常感谢，就是鼠标右键菜单如 果能汉化的话就完美了

### DISCUSSION THREAD

**Discussion by itv3** [RipplePiam/MobaXterm-Chinese-Simplified#8] [Q&A]
客格笔记发的以前版本的汉化，是包括鼠标右键菜单，如果能一起汉化的话就完美了

**Comment by RipplePiam** [comment]
Hi, 参考 #9 ：
鼠标右键菜单亦属于二进制文件编译好的，这部分内容若要汉化的话涉及到反编译，这有违此项目的初衷。
因此这部分暂不考虑去汉化。
非常感谢你的使用与建议。
